tests/openwisp2/settings.py

```diff
@@ -147,6 +148,14 @@
     CELERY_TASK_EAGER_PROPAGATES = True
     CELERY_BROKER_URL = 'memory://'
 
+CELERY_BEAT_SCHEDULE = {
+    'delete_old_notifications': {
+        'task': 'openwisp_notifications.tasks.delete_old_notifications',
+        'schedule': timedelta(days=1),
+        'args': None
```

Comment:
here we can pass the number of days @pandafy we may not need to add a new setting, we can try to keep it simple

Reply:
Okay! Sorry @RithikBanerjee :sweat_smile:

Reply:
Thats completely fine @pandafy
I am here to learn.